src/server.lisp

```diff
@@ -158,104 +158,104 @@ DISPATCH-TABLE and LOGGING-STREAM are both required arguments.  TIMEOUT is of ty
-                                                            :|error| (format nil "Method named \"~a\" is unknown."
-                                                                             (|RPCRequest-method| request))
-                                                            :|warnings| warnings)))
-                                (bt:timeout
-                                 (log-completion-message :err "Request ~a error: timed out"
-                                                         (|RPCRequest-id| request))
-                                 (setf reply (make-instance '|RPCError|
-                                                            :|id| (|RPCRequest-id| request)
-                                                            :|error| (format nil "Execution timed out.  Note: time limit: ~a seconds." timeout)
-                                                            :|warnings| warnings)))
-                                (otherwise
-                                 (log-completion-message :err
-                                                         "Request ~a error: Unhandled error in host program:~%~a"
-                                                         (|RPCRequest-id| request)
-                                                         c)
-                                 (setf reply (make-instance '|RPCError|
-                                                            :|id| (|RPCRequest-id| request)
-                                                            :|error| (princ-to-string c)
-                                                            :|warnings| warnings))))
-                              (go :skip-to-send)))
-                       (let ((#+sbcl sb-ext:*invoke-debugger-hook*
-                              #-sbcl *debugger-hook* #'error-processor))
-                         (setf start-time (get-internal-real-time))
-                         (setf request (deserialize raw-request))
-                         (unless (typep request '|RPCRequest|)
-                           (error 'not-an-rpcrequest
-                                  :object request))
-                         (cl-syslog:format-log logger ':info
-                                               "Request ~a received for ~a"
-                                               (|RPCRequest-id| request)
-                                               (|RPCRequest-method| request))
-                         (let ((kwargs-as-plist
-                                 (loop :for key :being :the :hash-keys :of (|RPCRequest-params| request)
-                                         :using (hash-value val)
-                                       :unless (string= "*args" key)
-                                         :append (list (str->lisp-keyword key) val)))
-                               (args-as-list (gethash "*args" (|RPCRequest-params| request))))
-                           (let ((f (gethash (|RPCRequest-method| request) dispatch-table)))
```

Comment:
I fused these `let`s to remove two extra levels of indentation, but this purely cosmetic and could be dropped.